src/python/pants/bin/BUILD

```diff
@@ -1,7 +1,10 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources(sources=["*.py", "!pants_loader.py"])
+python_sources(
+    sources=["*.py", "!pants_loader.py"],
+    dependencies=["src/python/pants/init:plugins"],
```

Comment:
Should this use `overrides`?

Reply:
It's not clear which file it would be attached to. But actually, it looks like `src/python/pants/BUILD` might be the best place for this dep.

Reply:
Another thing to consider is `pants_integration_test.py`, which I think still does need this dep

Reply:
Yea, I moved back to draft because there will likely be a bunch of tests which need fixing.